src/app/components/menu/Menu.jsx

```diff
@@ -3,45 +3,62 @@ import Box from '@mui/material/Box';
 import Drawer from '@mui/material/Drawer';
 import Button from '@mui/material/Button';
 import List from '@mui/material/List';
-import Divider from '@mui/material/Divider';
 import ListItem from '@mui/material/ListItem';
 import ListItemButton from '@mui/material/ListItemButton';
 import ListItemIcon from '@mui/material/ListItemIcon';
 import ListItemText from '@mui/material/ListItemText';
-import InboxIcon from '@mui/icons-material/MoveToInbox';
+import ShoppingCartIcon from '@mui/icons-material/ShoppingCart';
+import MonetizationOnIcon from '@mui/icons-material/MonetizationOn';
+import WidgetsIcon from '@mui/icons-material/Widgets';
 import MailIcon from '@mui/icons-material/Mail';
 import MenuIcon from '@mui/icons-material/Menu';
+import useSWR from 'swr';
 
-export default function TemporaryDrawer() {
+const fetcher = (url) => fetch(url).then((res) => res.json());
```

Comment:
generalnie pierwszy parametr useSWR to jest cache key. Nie musi być URLem a jest zwyczajnie identyfikatorem danego requesta. 

Zrobiłbym osobny plik gdzie po prostu masz: 

service.js 

``` 
async function fetchPosts() {
  // tutaj cała logika tego requesta 
}

// i potem tylko 

useSWR('posts', fetchPosts) 
```